draft-ietf-quic-transport.md

```diff
@@ -3157,30 +3158,35 @@ after a connection is established and 1-RTT keys are available; see
 
 ## Protected Packets {#packet-protected}
 
-All QUIC packets except Version Negotiation packets use authenticated encryption
-with associated data (AEAD) {{!RFC5116}} to provide confidentiality and
-integrity protection.  Retry packets use AEAD to provide integrity protection.
-Details of packet protection are found in {{QUIC-TLS}}; this section includes an
-overview of the process.
-
-Initial packets are protected using keys that are statically derived. This
-packet protection is not effective confidentiality protection.  Initial
-protection only exists to ensure that the sender of the packet is on the network
-path. Any entity that receives the Initial packet from a client can recover the
-keys necessary to remove packet protection or to generate packets that will be
+QUIC packets have different levels of cryptographic protection based on the
+type of packet. Details of packet protection are found in {{QUIC-TLS}}; this
+section includes an overview of the protections that are provided.
+
+Version Negotiation packets have no cryptographic protection; see
+{{QUIC-INVARIANTS}}.
+
+Retry packets use an authenticated encryption with associated data (AEAD;
+{{?AEAD=RFC5116}}) function to protect against accidental modification.
+
+Initial packets use an AEAD with keys that are statically derived. This does
+not provide effective confidentiality protection. Initial protection exists to
+ensure that the sender of the packet is on the network path. Any entity that
+receives the Initial packet from a client can recover the keys that will allow
+them to read the contents of packets or generate packets that will be
 successfully authenticated.
 
 All other packets are protected with keys derived from the cryptographic
 handshake. The type of the packet from the long header or key phase from the
 short header are used to identify which encryption keys are used. Packets
-protected with 0-RTT and 1-RTT keys are expected to have confidentiality and
-data origin authentication; the cryptographic handshake ensures that only the
+protected with 0-RTT and 1-RTT keys are expected to have strong confidentiality
+and integrity protection; the cryptographic handshake ensures that only the
```

Comment:
Maybe "authentication, confidentiality, and integrity protection"? I think that would be more consistent with the use of terms thatwe see in RFC 8446 section 1.

Reply:
Those are inclusive of other properties provided by TLS, so the packet protection doesn't strictly provide authentication in the same way.

Reply:
I'm not against viewing that way, though then I think it might be better to talk about 0-RTT, Handshake and 1-RTT packets, rather than just 0-RTT and 1-RTT packets.

It might be a good idea to change the first sentence of the paragraph to the following and remove the last sentence: _All other packets are expected to provide strong confidentiality and integrity protection using the keys derived from the cryptograph handshake; the handshake ensures that only the communicating endpoints receive the corresponding keys._